tests/interpreters/test_wgpu.py

```diff
@@ -13,7 +13,7 @@
 
 pytest.importorskip("wgpu", reason="wgpu is an optional dependency")
 
-from xdsl.interpreters.experimental.wgpu import WGPUFunctions  # noqa: E402
+from xdsl.interpreters.wgpu import WGPUFunctions  # noqa: E402
```

Comment:
What's this lint for? Realise it's unchanged but could there be an explanation in the comment